src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -317,6 +281,21 @@
       </allowed-values>
     </constraint>
   </define-assembly>
+  <define-assembly name="impact">
+    <formal-name>Impact Level</formal-name>
+    <description>The expected level of impact resulting from the described information.</description>
```

Comment:
So one of the things we will lose from the more modular code is the same `description` for the three different subsets of impact level. Personally, this does not bother me, but I think in terms of importance for that unique documentation per impact level is something we lose with the `assembly/@ref` approach. Are we cool with that at a high-level, @david-waltermire-nist, if I am happy with the other changes made here?

They look good so far. Will test locally on my workstation and have a deep think about other implications, but it seems low risk, good reward.

Reply:
I added the formal-names and descriptions back since this was added as a Metaschema feature.